xml/System.Security.Policy/Evidence.xml

```diff
@@ -209,11 +209,7 @@
           <AttributeName Language="C#">[System.Obsolete]</AttributeName>
           <AttributeName Language="F#">[&lt;System.Obsolete&gt;]</AttributeName>
         </Attribute>
-        <Attribute FrameworkAlternate="dotnet-plat-ext-6.0;net-6.0">
-          <AttributeName Language="C#">[System.Obsolete("This constructor is obsolete. Use the constructor which accepts arrays of EvidenceBase instead.")]</AttributeName>
-          <AttributeName Language="F#">[&lt;System.Obsolete("This constructor is obsolete. Use the constructor which accepts arrays of EvidenceBase instead.")&gt;]</AttributeName>
-        </Attribute>
-        <Attribute FrameworkAlternate="netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8">
+        <Attribute FrameworkAlternate="dotnet-plat-ext-6.0;net-6.0;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8">
```

Comment:
I'm not sure why the specific obsoletion messages got lost here for .NET Framework monikers.